src/api/index.md

```diff
@@ -2165,6 +2165,16 @@ type: api
   })
   ```
 
+  Sort using two keys:
+
+  ``` html
+  <ul>
+    <li v-for="user in users | orderBy ['lastName', 'firstName']">
```

Comment:
https://github.com/vuejs/vue/commit/ef555820c9f04d1695c632fae01f6bc409c32e43 (sorry I had to change a lot of your code)